src/nps/userSurvey.ts

```diff
@@ -250,6 +273,32 @@ class UserSurveyWebviewController extends ReactWebviewPanelController<
     }
 }
 
+export function getGithubIssueText(
+    comments: string,
+    extensionVersion: string,
+): string {
+    return `**Describe issue:**
+${comments}
+
+**Steps to Reproduce:**
+1.
+2.
+3.
+
+**Expected Behavior:**
+
+
+**Actual Behavior:**
+
+
+----
+|Software|Version|
+|--|--|
+|MSSQL Extension|${extensionVersion}|
+|VS Code|${vscode.version}|
+|OS|${os.type()} ${os.release()}|`;
```

Comment:
I wonder if it's worth adding a tag here, like "from-feedback-form".  So we can see what issues are coming in from this channel?  Not critical, since we need to look at all the issues.  But could be interesting to see which bugs are tied to NPS data.  Though I guess we would need to think through privacy concerns since we'd have two quasi-correlated input streams.

Reply:
I've updated it to automatically include the "user-filed" tag (as well as "Triage: needed").

Yup, let's definitely think through the privacy aspect.  I'll spin up a thread.

Reply:
Based on the email thread, both streams are GDPR compliant, so together they're supposed to be okay.  Users have full control over what they submit as a github issue.